### ISSUE TITLE
Documentation: parasites (Trac #956)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#950]
Documentation is needed for parasitic metaproject extensions.

Migrated from https://code.icecube.wisc.edu/ticket/956
```json
{
    "status": "closed", 
    "changetime": "2019-01-11T23:58:16", 
    "description": "Documentation is needed for parasitic metaproject extensions.", 
    "reporter": "richman", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1547251096686449", 
    "component": "cmake", 
    "summary": "Documentation: parasites", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2015-04-30T23:09:48", 
    "milestone": "", 
    "owner": "jvansanten", 
    "type": "enhancement"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150810T22:37:09: olivas changed milestone from "" to "offline V15-08-00"

**Comment by icecube-trac** [comment]
Trac update at 20150812T21:17:47: 
* jvansanten changed status from "new" to "closed"
* jvansanten commented:

> In changeset 2212:
> 
> ```text
> #!CommitTicketReference repository="IceTray" revision="2212"
> Document parasitic build system. Closes #950.
> 
> Along the way, mention the proper tarball name for use with icetray-start, and
> make parasitic build work on OSX.
> ```


* jvansanten changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20160318T21:13:59: 
* olivas commented:

> Milestone offline V15-08-00 deleted


* olivas changed milestone from "offline V15-08-00" to ""

**Comment by icecube-trac** [comment]
Trac update at 20190111T23:58:16: nwhitehorn commented:

> In changeset 170086:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="170086"
> Document parasitic build system. Closes #950.
> 
> Along the way, mention the proper tarball name for use with icetray-start, and
> make parasitic build work on OSX.
> ```
